docs/endpoints.rst

```diff
@@ -262,6 +262,23 @@ following:
    $ sudo systemctl restart my-globus-compute-endpoint.service
 
 
+AMQP Port
+---------
+
+.. TODO: fill in actual release number
+
+Endpoints receive tasks and communicate task results over AMQP. As of vX.X.X, that is
+done over port 443 by default, since firewall rules usually leave it open. In case 443
+is not open on a particular cluster, the port to use can be changed in the endpoint
+config via the ``amqp_port`` option, like so:
+
+.. code-block:: yaml
+
+  amqp_port: 5671
+
+Note that only ports 5671, 5672, and 443 are supported with the Compute hosted services.
```

Comment:
Useful, but context and indentation in YAML are ... "everything."  Suggest including another top-level item (`display_name`?) for a mental anchor of where this configuration resides.

Reply:
Yes I second what Kevin said.  Whenever I configure something new the hardest part is usually figuring out which directory, file, and which section(indentation) a config variable goes.  (and string vs. number)

Reply:
Good call, was debating whether to do this myself 👍